packages/app/src/cli/models/extensions/specifications/flow_template.ts

```diff
@@ -0,0 +1,85 @@
+// import {createUIExtensionSpecification} from '../ui.js'
+import {BaseSchema} from '../schemas.js'
+import {createExtensionSpecification} from '../specification.js'
+import {loadLocalesConfig} from '../../../utilities/extensions/locales-configuration.js'
+import {joinPath} from '@shopify/cli-kit/node/path'
+import {zod} from '@shopify/cli-kit/node/schema'
+import {AbortError} from '@shopify/cli-kit/node/error'
+import {glob} from '@shopify/cli-kit/node/fs'
+import fs from 'fs'
+
+const FlowTemplateExtensionSchema = BaseSchema.extend({
```

Comment:
We have a [PR](https://github.com/Shopify/core-build-north-star-app/pull/9) for our extensions shape in the North Star repo. IDK if you want to add your template extension there and validate the extra fields with DC. There might be a need for alignment there LMK if you need help with that.